internal/manifests/targetallocator/container_test.go

```diff
@@ -211,36 +209,6 @@ func TestContainerHasEnvVars(t *testing.T) {
 	assert.Equal(t, expected, c)
 }
 
-func TestContainerHasProxyEnvVars(t *testing.T) {
```

Comment:
Why are we removing this test?

Reply:
We don't set up proxy environmental variables.